src/frontend/get-jira-configuration.ts

```diff
@@ -91,7 +100,7 @@ export default async (
 
 		const failedConnections = getFailedConnections(
 			installations,
```

Comment:
This is where the real problem lies: the `getInstallation()` doesn't return a type and it's also return 2 different types in one function instead of having one resolved type and another rejected type.  Fixing `getInstallation` would make life much easier as to find the failed installations, you just need to find those that were rejected.

Reply:
I think this be done as a separate ticket. My follow-up item was to add typings to `getFailedConnections`, not refactor a function that was a part of the original codebase when we took over.

Reply:
yeah, fair.